src/ray/protobuf/common.proto

```diff
@@ -563,14 +563,78 @@ message PlacementGroupSpec {
   double max_cpu_fraction_per_node = 10;
 }
 
+// Unit of scheduling. A VC Bundle can be allocated to a single (physical) node with
+// certain resources. A task/actor in that VC can use those resources.
 message VirtualClusterBundle {
+  // Allocates e.g. {CPU: 1.0} from the physical cluster to this virtual cluster.
+  // Those resource amounts are fixed.
   map<string, double> resources = 1;
+  // Labels added to the node that scheduled this bundle.
+  map<string, string> labels = 2;
+  // If true, for a resource name *not* defined in `resources`, a task/actor can request
+  // and the virtual cluster would try to acquire them from the physical cluster.
+  bool allow_wildcard_resources = 3;
 }
 
-// Specification of a virtual cluster. Each Bundle is scheduled atomically to 1 node.
+message VirtualClusterBundleSet {
+  VirtualClusterBundle bundle = 1;
+  // Initial scheduling will make at least this many replicas.
+  int64 min_replicas = 2;
+  int64 max_replicas = 3;
+}
+
+// Specification of a virtual cluster.
+// A Virtual Cluster can be defined with a range of numbers of a certain shaped bundles,
+// and optionally can request resource names not specified in any bundles.
+// For example:
+// 1) Default VC if nothing is specified, can eat all resources in physcial cluster
```

Comment:
Instead of allow_wildcard_resources, I thought the default behavior would be to create one default VC that is the entire cluster, and when new VCs get created, we subtract resources from the default VC?